mne/viz/_3d.py

```diff
@@ -301,10 +302,15 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
         If True, plot MEG sensors as points in addition to showing the helmet.
     eeg_sensors : bool
         If True, plot EEG sensors as points.
-    dig : bool
-        If True, plot the digitization points.
+    dig : bool | 'fiducials'
+        If True, plot the digitization points. "fiducials" to plot fiducial
+        points only.
     ref_meg : bool
         If True (default False), include reference MEG sensors.
+    fiducials : bool | str
```

Comment:
this should probably be `mri_fiducials`. The digitized fiducials are already displayed as part of `dig`